src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <section class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
+        <input
+          class="search-bar__search search-bar__search--large"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="search-bar search-bar__small"
+        data-qa="small"
+      >
+        <input
+          class="search-bar__search search-bar__search--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The same issue is here with the class naming. Consider renaming `search-bar__small` to `search-bar--small` to indicate that it is a modifier of `search-bar`.